documentation/modules/proc-accessing-kafka-using-loadbalancers.adoc

```diff
@@ -36,27 +43,28 @@ spec:
 
 . Create or update the resource.
 +
-This can be done using `kubectl apply`:
 [source,shell,subs=+quotes]
 kubectl apply -f _KAFKA-CONFIG-FILE_
++
+`loadbalancer` type services and loadbalancers are created for each Kafka broker, as well as an external _bootstrap service_.
+The bootstrap service routes external traffic to the Kafka brokers.
+DNS names and IP addresses used for connection are propagated to the `status` of each service.
 
-. Find the hostname of the bootstrap loadbalancer.
+. Retrieve the address of the bootstrap service to access the Kafka cluster.
```

Comment:
This whole section / bullet point should be replaced with something like this

```suggestion
. Retrieve the address of the bootstrap service to access the Kafka cluster.

[source,shell,subs=+quotes]
kubectl get kafka _KAFKA-CLUSTER-NAME_ -o=jsonpath='{.status.listeners[?(@.type=="external")].bootstrapServers}{"\n"}'
```

Reply:
@scholzj I originally changed this and then changed back, thinking the _status_ was the key. I've replaced now, but we don't want both?

Reply:
What we now basically do is we do the original steps in the operator and put it always into a single place. So instead of taking it from the status of the operator, the user can take it from the Kafka status with all the methods. So the user does not have to search for the services etc.

Reply:
Thanks. I get it now.